src/metabase/db/data_migrations.clj

```diff
@@ -178,7 +167,7 @@
 (defmigration ^{:author "senior", :added "0.30.0"} clear-ldap-user-local-passwords
   (db/transaction
     (doseq [user (db/select [User :id :password_salt] :ldap_auth [:= true])]
-      (db/update! User (u/the-id user) :password (creds/hash-bcrypt (str (:password_salt user) (UUID/randomUUID)))))))
+      (db/update! User (u/the-id user) :password (creds/hash-bcrypt (str (:password_salt user) (java.util.UUID/randomUUID)))))))
```

Comment:
Is this change intentional? (And if so, why?) Seems like it's done in every one of your migration PRs.

Reply:
TBH I only did it because the new version of `clj-refactor` keeps trying to change `(:import java.util.UUID)` to `(:import (java.util UUID))` when you run `cljr-clean-namespace` and I got sick of changing it back by hand in 8 different branches. It gets removed anyway in the end